docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -31,6 +44,26 @@ To follow this guide, you need:
 - Docker
 - The docker-compose plugin
 
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Quick start for Tempo](https://killercoda.com/grafana-labs/course/tempo/quick-start).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/tempo/tempo-ile.svg)
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
+{{< /admonition >}}
+
+<!-- INTERACTIVE ignore END -->
+
+{{<docs/ignore>}}
```

Comment:
Normally we use and admonition shortcode. Do we need to remove any shortcodes for Killercode?